0000-template.md

```diff
@@ -55,16 +57,17 @@ Why should we *not* do this?
 - What other designs have been considered and what is the rationale for not choosing them?
 - What is the impact of not doing this?
 
-# Prior art
+#prior art
 [prior-art]: #prior-art
 
-Discuss prior art, both the good and the bad, in relation to this proposal.
-A few examples of what this can include are:
+Discuss prior art, the good and the bad, in relation to this proposal.
+Here are some examples of what this might include:
 
-- For language, library, cargo, tools, and compiler proposals: Does this feature exist in other programming languages and what experience have their community had?
-- For community proposals: Is this done by some other community and what were their experiences with it?
+- For language, library, cargo, tool and compiler proposals: Does this feature exist in other programming languages and what has been the experience of their community?
+- For kernel proposals: Does this functionality exist in other kernels and what has been the experience of their community?
```

Comment:
This part is basically the same as the next one and means the same thing, you should remove it